src/main/java/string/Exer10.java

```diff
@@ -0,0 +1,16 @@
+package string;
+
+public class Exer10 {
+
+    public int sumNumbers(String num1, String num2) {
+        return getNumber(num1) + getNumber(num2);
+    }
+
+    public int getNumber(String number) {
```

Comment:
Giống bài 11, tính toán trên số lớn nên chỉ dùng string nha

Reply:
fixed